bin/peterborough-layer-update

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+source /data/mysociety/shlib/deployfns
+read_conf "$(dirname "$0")/../conf/general.yml"
+
+TMPDIR=$(mktemp -d) || exit 1
+trap 'rm -rf "$TMPDIR"' EXIT
+cd $TMPDIR
+
+curl -sS -O -u $OPTION_peterborough__username:$OPTION_peterborough__password \
+    "sftp://${OPTION_peterborough__host}${OPTION_peterborough__dir}/trees_invent.zip"
+unzip trees_invent.zip
+mv *.{dbf,prj,shp,shx} $OPTION_peterborough__out
```

Comment:
It'd be nice to have some checking here that it's not going to overwrite working files with some 0 byte or corrupt entries or something. Not sure (besides a size check) what else you could do - perhaps run `ogrinfo -ro -so` and check its output in some way?

Reply:
Have pushed ad94163 which uses ogrinfo's exit code to determine whether the shapefiles are valid or not.